package.json

```diff
@@ -37,9 +37,9 @@
     "babelify": "^7.3.0",
     "browserify": "^16.2.3",
     "http-server": "^0.10.0",
-    "vue": "^2.5.17"
+    "vue": "^3.1.5"
   },
   "dependencies": {
-    "chartist": "^0.11.0"
+    "chartist": "git+https://github.com/crisp-dev/chartist-js.git#59cdecf7ad3fa12a6a18facf7ca34dc3e720bcb5"
```

Comment:
You can use the regular version instead

Reply:
So this change is not necessary?

Reply:
It's not. We use a fork for our app because we patched some bugs with Chartist. Those bugs were not merged yet in chartist. 

It's not Vue-related at all, so it's safe t use official chartist instead.